src/components/Accordion/components/Header.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+import { Colors } from '../../../essentials';
+import { Box } from '../../Box/Box';
+
+export const Header = styled(Box).attrs({ p: '2', color: Colors.AUTHENTIC_BLUE_900 })`
```

Comment:
_No change necessary at the moment (but welcome)_: In an effort to make use of the new semantic colors we introduced, I wanted to let you know about another way to use this variable in the code. It's what we want to encourage moving forward: Semantic/Functional colors. Instead of accessing the `Colors` variable, you will use the `get` function in the `themeGet.ts` file in this project to get a semantic color from our library. See reference here: https://wave.free-now.com/essentials/colors#functional-reference.

```
color: get('semanticColors.text.primary')
```